sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -0,0 +1,340 @@
+## API Report File for "@azure/attestation"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+
+// @public
+export type AttestationAttestOpenEnclaveResponse = AttestationResponse & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: AttestationResponse;
+    };
+};
+
+// @public
+export type AttestationAttestSgxEnclaveResponse = AttestationResponse & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: AttestationResponse;
+    };
+};
+
+// @public
+export type AttestationAttestTpmResponse = TpmAttestationResponse & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: TpmAttestationResponse;
+    };
+};
+
+// @public
+export interface AttestationCertificateManagementBody {
+    policyCertificate?: JsonWebKey;
+}
+
+// @public (undocumented)
+export class AttestationClient extends AttestationClientContext {
+    constructor(instanceUrl: string, options?: AttestationClientOptionalParams);
+    // Warning: (ae-forgotten-export) The symbol "Attestation" needs to be exported by the entry point index.d.ts
```

Comment:
forgotten types

Reply:
Manually fixed for this PR. Filed Issue https://github.com/Azure/autorest.typescript/issues/808